plugin.audio.somafm/addon.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.somafm"
+       name="SomaFM"
+       version="2.0.1"
+       provider-name="Soma-FM-Kodi-Add-On Organization">

```

Comment:
The original add-on is from @Oderik, so he/she should approve for this Matrix version.

Reply:
@basrieter The original addon author (@Oderik) doesn't maintain the previous version of this addon anymore (see: https://github.com/Oderik/xbmc-somafm/issues/13#issuecomment-852879925)

But granted us permission to take over their code base and run it under and organisation GitHub account (see: https://github.com/Oderik/xbmc-somafm/issues/13#issuecomment-970142899)

What other kind of permission would be required?